Correction orthographique

helmasaur opened the pull request:
Il manquait l'accent circonflexe à *s'entraîner*.

| Q                                   | R
| ----------------------------------- | -------------------------------------------
| Type de modification                | correction
| Ticket(s) (_issue(s)_) concerné(s)  | 

### QA
- [x] Ça fonctionne !
- [x] Code relu et approuvé !

helmasaur committed:
Correction orthographique

Il manquait l'accent circonflexe à *s'entraîner*.

gllmc commented:
Salut,

Avec la [nouvelle orthographe](https://www.orthographe-recommandee.info/enseignement/regles.pdf), ce n'est plus nécessaire. ;)

helmasaur commented:
Je préfère personnellement avec l'accent circonflexe mais dans ce cas là, il faut une cohérence entre les lignes [12](https://github.com/Helmasaur/zds-site/blob/c7ad80876a8e6c5a3ad1713daa6763028d5f0330/templates/forum/index.html#L12) et [33](https://github.com/Helmasaur/zds-site/blob/c7ad80876a8e6c5a3ad1713daa6763028d5f0330/templates/forum/index.html#L33). C'était surtout pour cette raison que j'ai fait la modification.

SpaceFox commented:
Si c'est une question de cohérence, [autant utiliser la « nouvelle » orthographe](https://textes.spacefox.fr/rédiger/l-orthographe-et-la-réforme-de-1990/).

helmasaur commented:
Je n'aime pas vraiment la nouvelle orthographe donc à vous de décider ce qui est retenu. En tout cas, l'incohérence est relevé (: .

SpaceFox commented:
C'est pas une question de « aimer » ou pas en fait, cf l'argumentation derrière mon lien.

helmasaur commented:
Les deux orthographes sont considérées bonnes donc, si, il s'agit d'une question de préférence. Dans tous les cas, ce n'est pas à moi de faire le choix. Il faudrait juste choisir la même orthographe pour les deux lignes.

SpaceFox commented:
Oui et non : il n'y a aucune raison _logique_ de conserver l'ancienne, seulement des considérations esthétiques ou un refus du changement. Donc, simplifions et utilisons la nouvelle.

vhf commented:
100% d'accord. Nouvelle partout ancienne nulle part.

helmasaur committed:
Orthographe rectifiée

Modification de *s'entrainer* à la ligne 12 et 33 afin que ceux deux lignes soit écrites avec l'orthographe rectifiée.

helmasaur commented:
J'ai modifié la ligne [33](https://github.com/Helmasaur/zds-site/blob/c7ad80876a8e6c5a3ad1713daa6763028d5f0330/templates/forum/index.html#L33) pour qu'elle prenne l'orthographe rectifiée, comme souhaité.

gllmc commented:
Merci @Helmasaur !
